client/client.go

```diff
@@ -293,3 +295,38 @@ func (lc *LogClient) postAndParse(path string, body []byte, rsp interface{}) (*H
 	}
 	return httpData, nil
 }
+
+// AddChain performs an add-chain request, posting the provided certificate
+// chain to the CT Log hosted at LogClient.url.  The first certificate in chain
+// should be the end-entity certificate, with the second chaining to the first
```

Comment:
should -> must

Monologue will only ever post chains that start with an end-entity cert, right?  It won't post chains of CA certs?